pkg/controller/credentialsrequest/credentialsrequest_controller.go

```diff
@@ -255,7 +255,7 @@ type ReconcileCredentialsRequest struct {
 // +kubebuilder:rbac:groups=core,resources=secrets;configmaps;events,verbs=get;list;watch;create;update;patch;delete
 // +kubebuilder:rbac:groups=core,resources=namespaces,verbs=get;list;watch
 // +kubebuilder:rbac:groups=config.openshift.io,resources=clusterversions,verbs=get;list;watch
-// +kubebuilder:rbac:groups=config.openshift.io,resources=infrastructures,verbs=get;list;watch
+// +kubebuilder:rbac:groups=config.openshift.io,resources=infrastructures;dnses,verbs=get;list;watch
```

Comment:
We should get rid of these comments, since they’re no longer being used to generate rbac and changing them has no effect on anything.

Reply:
the CCO repo still generates them. i added 'dnses' b/c this PR missed adding it https://github.com/openshift/cloud-credential-operator/pull/105, and my local builds were backing out the manual addition of 'dnses' from that PR.

Reply:
> the CCO repo still generates them

Not sure what this means? :) But afaik the only thing that adds these is kubebuilder when you add a new type/controller, but it adds it to the generated file, not to existing files. My concern is that going forward we may get the idea that adding permissions here will be enough without realizing that we really need to add them [here](https://github.com/openshift/cloud-credential-operator/blob/master/manifests/01_deployment.yaml#L15)

Reply:
I mean that the Makefile in the CCO repo still auto-generates the manifests/01_deployment.yaml.

I think we're both worried about the same thing :). Right now in the master branch if you run 'make' you end up with this diff
```
[jdiaz@minigoomba cloud-credential-operator (master *$%=)]$ git diff manifests/01_deployment.yaml
diff --git a/manifests/01_deployment.yaml b/manifests/01_deployment.yaml
index e278a4e5..2c31d450 100644
--- a/manifests/01_deployment.yaml
+++ b/manifests/01_deployment.yaml
@@ -62,7 +62,6 @@ rules:
   - config.openshift.io
   resources:
   - infrastructures
-  - dnses
   verbs:
   - get
   - list
```

In order to not lose the 'dnses' entry that's already in the master branch of manifests/01_deployment.yaml, I added 'dnses' so that when 'make manifests' runs it keeps 'dnses' in the generated 01_deployment.yaml.

Reply:
Sorry just looked at the Makefile, I see what you mean. For some reason I thought we’d abandoned the rbac generated from the code.